platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/events/ConsensusData.java

```diff
@@ -110,14 +113,7 @@ public boolean equals(final Object o) {
 
     @Override
     public int hashCode() {
-        return new HashCodeBuilder(17, 37)
-                .append(generation)
-                .append(roundCreated)
-                .append(stale)
-                .append(roundReceived)
-                .append(consensusOrder)
-                .append(consensusTimestamp)
-                .toHashCode();
+        return Objects.hash(generation, roundCreated, stale, consensusTimestamp, roundReceived, consensusOrder);
```

Comment:
I assume the order change here is benign and that the hash value only needs to be the same per lifetime of the running app and not across multiple runs.

Reply:
The hashcode produced will be different regardless of order; `Objects.hash` uses different constants than `HashCodeBuilder`.  `HashCodeBuilder` also handles structures like arrays differently than `Objects.hash`.